tools/commands/build_example.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import re
+import subprocess
+import sys
+import os
+import commands.command_utils as command_utils
+
+
+def parse_args(args):
+    parser = command_utils.get_options_parser(plugins=True, exclude=True, run_on_changed_packages=True, base_sha=True)
+    return parser.parse_args(args)
+
+
+def _build_examples(plugin):
+    print(f'============= Building for {plugin} =============')
+    example_dir = os.path.join(plugin, 'example')
+    subprocess.run('flutter-tizen pub get',
+                   shell=True,
+                   cwd=example_dir,
+                   stderr=subprocess.PIPE,
+                   stdout=subprocess.PIPE)
+
+    completed_process = subprocess.run('flutter-tizen build tpk --device-profile wearable -v',
+                                       shell=True,
+                                       cwd=example_dir,
+                                       universal_newlines=True,
+                                       stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    print(completed_process.stdout)
```

Comment:
Removing `subprocess.PIPE` shows logs in CI terminal in realtime.

```suggestion
    completed_process = subprocess.run('flutter-tizen build tpk --device-profile wearable -v',
                                       shell=True,
                                       cwd=example_dir)
```